### PR TITLE
Modified Source Files

### PR DESCRIPTION
**Pull request by an-rahulpandey**

### CODE REVIEW

**Comment by an-rahulpandey** [comment]
have you seen the code?

File: www/iRate.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+cordova.define("me.rahul.plugins.iRate.iRate", function(require, exports, module) { var cordova = require('cordova'),
```
Comment (aogilvie) [inline www/iRate.js]: You do not need to include this line. CLI install will wrap all the JavaScript automatically.

**Comment by aogilvie** [comment]
Sorry, just had the opportunity to check this.

Unfortunately it breaks during CLI plugman install. Please check my comments to fix it.

Sample code used: 

```
cordova create myApp
cd myApp/
cordova platform add ios
cordova plugin add /path/to/phonegap-plugin-iRate
cordova prepare ios
```

**Comment by an-rahulpandey** [comment]
I installed the plugin now using my git url and it is working fine.

**Comment by an-rahulpandey** [comment]
Did you check the version again?

**Pull request merged**